lib/scrabble.rb

```diff
@@ -1,7 +1,17 @@
+require '../lib/player'
+
 class Scrabble
+  attr_reader :word
 
   def score(word)
-    1
+    @word = word
+    word = Scrabble.new("hello")
+    letters = word
```

Comment:
Remember that word will hold a string getting passed in from the outside. No need to make a new Scrabble instance or set it to another variable.  Take a look at my comment from line 14 to see some suggestions on what you might have wanted to do with your string.